### ISSUE TITLE
Object detection API not detecting objects in images of different size than training set

### ISSUE THREAD

**Issue by marcoistasy** [tensorflow/models#7388]
### System information
- **What is the top-level directory of the model you are using**: N/A
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes. The exact details are explained below.
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: OSX 10.14.5
- **TensorFlow installed from (source or binary)**: Binary
- **TensorFlow version (use command below)**: 1.13.1
- **Bazel version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: N/A
- **GPU model and memory**: N/A
- **Exact command to reproduce**: See below

### Describe the problem
I have trained my model on several [test images](https://github.com/marcoistasy/imagineOCR/tree/develop/imagineOCR/example_data/images) and have trained a `rfcn_resnet101_coco` model to detect the letter *o* in these images.

I thought the model was working fine as it was detecting *o* (with decent accuracy) in the images I was passing to it. Please see [here](https://github.com/marcoistasy/imagineOCR/blob/develop/imagineOCR/example_data/test/results/image2_result.png) and [here](https://github.com/marcoistasy/imagineOCR/blob/develop/imagineOCR/example_data/test/results/image3_result.png) for examples.

However, when I passed in two more images whose size varied greatly from the first two, the model was unable to detect anything! For example, when I passed in a [full page](https://github.com/marcoistasy/imagineOCR/blob/develop/imagineOCR/example_data/test/results/image1_result.png) or a [single world](https://github.com/marcoistasy/imagineOCR/blob/develop/imagineOCR/example_data/test/results/image4_result.png) to the model, the output was the same as the input.

I believe this is because the last two images I passed in where much larger or smaller than the [original training set](https://github.com/marcoistasy/imagineOCR/tree/develop/imagineOCR/example_data/images) that I used to train the model, whereas the first two or roughly the same size. 

I am wondering if there is anyway to train my model such that it can detect the letter *o* in images of varying sizes and not just those similar to the training input. I thought of tweaking my `pipeline.config` (found [here](https://github.com/marcoistasy/imagineOCR/blob/develop/imagineOCR/example_data/model/pipeline.config)) but am not sure where to start. Alternatively, this might be a bug in the `object_detection` source code. 

### Source code / logs
Here is a link to the Github repo where the [source code](https://github.com/marcoistasy/imagineOCR/tree/develop/imagineOCR/example_data) is held.

**Comment by tombstone** [comment]
@marcoistasy Sorry for the delay. We can use stronger data augmentation during training to address such issues. One option is to use ssd_random_crop. I can't access your config at the moment. See our ssd configs for examples of using these data augmentation options.
